generated/1.26/crds/config.supervisor.pinniped.dev_federationdomains.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.17.1
```

Comment:
These diffs look wrong... it appears to be a downgrade.

Reply:
Agreed but that's apparently the current state of the codegen: https://ci.pinniped.dev/teams/main/pipelines/pull-requests/jobs/verify-codegen/builds/220 fails for 1.26 and 1.29

Reply:
This was resolved after the CI jobs reran. I pushed a new commit to this PR.